src/main/java/edu/harvard/iq/dataverse/mydata/DataRetrieverAPI.java

```diff
@@ -384,7 +387,7 @@ public String retrieveMyDataAsJsonString(@QueryParam("dvobject_types") List<Stri
         List<String> filterQueries = this.myDataFinder.getSolrFilterQueries();
         if (filterQueries==null){
             logger.fine("No ids found for this search");
-            return this.getJSONErrorString(DataRetrieverAPI.MSG_NO_RESULTS_FOUND, null);
+            return this.getJSONErrorString(noMsgResultsFound, null);
```

Comment:
Should we do something similar in MyDataFinder (lines 214 and 281) where it gets an error message from the bundle using DataRetrieverAPI.MSG_NO_RESULTS_FOUND?

Reply:
Hey @JayanthyChengan, just wondering if you've had a chance to think about this. we'd like to get it merged along with your PR on large file internationalization. Thanks!

Reply:
Good Morning @sekmiller :  Please review the changes.  Thanks!